scripts/mosh.pl

```diff
@@ -382,6 +382,11 @@ sub predict_check {
   if ( scalar @command > 0 ) {
     push @server, '--', @command;
   }
+  
+  if ( $server cmp 'mosh-server' ) {
```

Comment:
Unfortunately, this will break the existing, common usage of `mosh  --server=/usr/local/bin/mosh-server`, so it isn't the right fix.  A workaround is to set up a `mosh-server` wrapper script on your server, but we should find a better fix for this problem.